### PR TITLE
139 licences

### PR DESCRIPTION
**Pull request by elinkronander**
- Description of the licences in the licence.md file
- added a licence topic in the reademe.md
- updated background and motivation in the readme.md
- added a reference on the life cycle index page

Did not change the licence link in the "about section" of the repo

### CODE REVIEW

File: content/data-life-cycle/_index.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ The research data life cycle can be divided into several phases as seen in the w
       <div class="card-body">
         {{< data-life-cycle >}}
       </div>
+      The reasearch data life cycle wheel is reused from <a href=https://rdmkit.elixir-europe.org/>RDMKit</a>
```
Comment (elinkronander) [inline content/data-life-cycle/_index.md:25]: I would have liked this text to be right-aligned and possibly a bit further down but I don't know how to do that so help is appreciated

File: LICENSE [hunk 2]
```diff
@@ -1,395 +1,46 @@
...
-granted the Licensed Rights in consideration of Your acceptance of
-these terms and conditions, and the Licensor grants You such rights in
-consideration of benefits the Licensor receives from making the
-Licensed Material available under these terms and conditions.
+The [data life cycle wheel](layouts/partials/data_life_cycle.html) is is reused under a CC-BY license from the [RDMKit](https://rdmkit.elixir-europe.org/)
 
+## Software
 
-Section 1 -- Definitions.
+Copyright (c) 2022 SciLifeLab
```
Comment (elinkronander) [inline LICENSE:28]: I copied this part from data platform repo but unsure if the 
Copyright (c) 2022 SciLifeLab should be kept or not?

File: content/data-life-cycle/_index.md (hunk 1 above)
Comment (annaasklof) [reply content/data-life-cycle/_index.md:25]: I'm on it 👍

Comment (YvonneKallberg) [reply content/data-life-cycle/_index.md]: RDMkit not RDMKit

File: content/data-life-cycle/_index.md [hunk 3]
```diff
@@ -13,18 +13,21 @@ menu:
...
+    </div>
+  </div>
+  <div class="col-md-1">
+    <div class="card-body">
+    </div>
+  </div>
+</div>
+<br><br><br>
+
+The reasearch data life cycle wheel is reused from [RDMKit](https://rdmkit.elixir-europe.org/).
```
Comment (elinkronander) [inline content/data-life-cycle/_index.md:33]:
```suggestion
The reasearch data life cycle wheel is reused from [RDMkit](https://rdmkit.elixir-europe.org/).
```

**Pull request merged**